src/jit/namedintrinsiclist.h

```diff
@@ -31,125 +31,4 @@ enum NamedIntrinsic : unsigned int
 #endif // FEATURE_HW_INTRINSICS
 };
 
-#if defined(FEATURE_HW_INTRINSICS) && defined(_TARGET_XARCH_)
```

Comment:
Nice! I always thought it was unfortunate that this was in this file.

Reply:
I agree 😄 

I think there is probably some more cleanup that could still happen, but that can be saved for another PR